Library/Formula/nsq.rb

```diff
@@ -72,6 +73,36 @@ def install
   end
 
   test do
-    system "#{bin}/nsqd", "--version"
+    lookupd = fork do
+      system bin/"nsqlookupd"
+    end
+    sleep 2
+    d = fork do
+      system bin/"nsqd", "--lookupd-tcp-address=127.0.0.1:4160"
+    end
+    sleep 2
+    admin = fork do
+      system bin/"nsqadmin", "--lookupd-http-address=127.0.0.1:4161"
+    end
+    sleep 2
+    to_file = fork do
+      system bin/"nsq_to_file", "--topic=test", "--output-dir=#{testpath}",
+             "--lookupd-http-address=127.0.0.1:4161"
+    end
+    sleep 2
+    curl = fork do
+      system "curl", "-d", "'hello world 2'", "http://127.0.0.1:4151/put?topic=test"
+    end
```

Comment:
`curl` doesn't need to be in a fork, but the rest [apparently do](http://nsq.io/overview/quick_start.html)—I tried a single fork and couldn't get the test to complete.